File: Waffle Studio/recruit/Problem 3/pr3_skel_java/Request.java
enum Command {
    ADD, DELETE, LIST, PIN, UNPIN, QUIT
}

enum Option {
    r, o, g, n, a
}

//< @brief String command 를 전달받아서, Application.execute()에서 사용될 수 있도록 적절히 파싱해 두는 클래스
public class Request {
    // TODO implement here: 필드 자유롭게 추가
    
    //< @brief 생성자
    Request(String command) {
        // TODO implement here
    }
}

File: Waffle Studio/recruit/Problem 3/pr3_skel_py/request.py
from enum import Enum

class Command(Enum):
    ADD = 0
    DELETE = 1
    LIST = 2
    PIN = 3
    UNPIN = 4
    QUIT = 5

class Option(Enum):
    r = 0
    o = 1
    g = 2
    n = 3
    a = 4

class Request:
    # TODO: fields/attributes

    # TODO: constructor
    def __init__(self, command):
        pass